src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -418,7 +418,8 @@ final class RiskProvider: RiskProviding {
 		let configuration = RiskCalculationConfiguration(from: appConfiguration.riskCalculationParameters)
 
 		let enfRiskCalculationResult = enfRiskCalculation.calculateRisk(exposureWindows: exposureWindows, configuration: configuration)
```

Comment:
Would it make sense to pass in the filtered windows already instead of filtering twice? In the tech spec filtering seems to be part of `Obtaining Exposure Windows`, not the Risk Calculation itself. So we could even think about returning a filtered result from `executeExposureDetection(...)` already.